packages/mesos/extra/upgrade_cni.py

```diff
@@ -0,0 +1,36 @@
+#!/opt/mesosphere/bin/python3
+
+import os
+import shutil
+import sys
+
+
+def main(old_dir, new_dir):
+    '''
+    Moves all the directories from old_dir to new_dir
+
+    @type old_dir: str, old CNI directory
+    @type new_dir: str, new CNI directory
+    '''
+    if not os.path.exists(old_dir):
+        return
+
+    for d in os.listdir(old_dir):
+        src = os.path.join(old_dir, d)
+        if os.path.isdir(src):
+            dst = os.path.join(new_dir, d)
+            shutil.move(src, dst)
```

Comment:
I think we should only move those ipam data dir that is owned by DC/OS. Namely
`dcos_overlay_network_default_name`
`dcos6` (we should make that a constant)
and `mesos-bridge` (we should make that a constant)

And we should not move other directories since they might not be owned by DC/OS. (Imagine the customer is also using some other system that depends on host local IPAM).